scania-digital-design-system/sdds-cms
# AB#562 Feature/migrate strapi stable

Migrate strapi from beta.18 to 3.5.3

How to test:

Follow this guide and create JWT secret
https://strapi.io/documentation/developer-docs/latest/update-migration-guides/migration-guides/migration-guide-3.0.x-to-3.1.x.html#_2-define-the-admin-jwt-token 

Update database local: 
- remove constraint "NOT_NULL" in table users-permissions_user and strapi_administrator column username
- Go to schema > table name > right click > properties > columns > turn off "not null" toggle in "username" column

Run npm i
Run npm run build -- --clean
Run npm run develop
Check in http://localhost:1340/admin/

Fix: #12 [AB#562](https://dev.azure.com/scaniadigitaldesignsystem/3d9eb9c6-0045-473d-bc1d-e842dd779200/_workitems/edit/562)
Close #24 #25
